src/node/config.ts

```diff
@@ -18,18 +19,53 @@ type RawConfig<ThemeConfig = unknown> =
   | Promise<UserConfig<ThemeConfig>>
   | (() => UserConfig<ThemeConfig> | Promise<UserConfig<ThemeConfig>>);
 
+/**
```

Comment:
If there is single line commemt, you could use `//` instead.

Reply:
When someone wants to see the description of the function through hover, `/**` can help him。
![image](https://user-images.githubusercontent.com/81058033/197926567-acde6219-fa07-4618-a0ff-7063ebc452a5.png)